app/src/types.ts

```diff
@@ -0,0 +1,81 @@
+import { Dispatch, SetStateAction } from 'react'
+import { UrlObject } from 'url'
+
+/*
+ * Types relating to persistent user data
+ */
+export type Clinic = {
+  id: number
+  agency: string
+  agencyAddress: string
+  agencyTelephone: string
+  clinic: string
+  clinicAddress: string
+  clinicTelephone: string
+  county: string
+  zip: string
+}
```

Comment:
Notes for reviewers: This represents a single unit of clinic data that is supplied in `/public/data/clinics.json` and matches the object structure of that data file.